resources/i18n/en.json

```diff
@@ -1119,7 +1119,11 @@
     "Installed": "Installed",
     "GithubInternalError": "Github Internal Error.",
     "GithubAPILimitError": "Github API Limitation Error.",
-    "GithubAPIEtcError": "Github Server Issue."
+    "GithubAPIEtcError": "Github Server Issue.",
+    "ImportGitlabRepo": "Import Gitlab Repository",
+    "GitlabRepoWillBeFolder": "Imported GitLab repositories are converted to folders and can be mounted when session starts.",
+    "GitLabURL": "GitLab URL",
+    "GitLabDefaultBranch": "Gitlab Branch Name"
```

Comment:
Let's be consistent with the naming for the messages: `Gitlab` or `GitLab`. Please choose one.

Reply:
I choose `Gitlab`.